app/models/ticket_purchase.rb

```diff
@@ -48,9 +44,21 @@ def self.purchase_ticket(conference, quantity, ticket, user)
   def self.update_quantity(conference, quantity, ticket, user)
     purchase = TicketPurchase.where(ticket_id: ticket.id,
                                     conference_id: conference.id,
-                                    user_id: user.id).first
+                                    user_id: user.id,
+                                    paid: 'f').first
 
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def self.update_paid_ticket_purchases(conference, user, payment)
+    paid_ticket_purchases = TicketPurchase.where(conference_id: conference.id,
+                                                 user_id: user.id,
+                                                 paid: 'f')
+    begin
+      paid_ticket_purchases.each do |ticket|
+        ticket.update_columns(paid: 't', payment_id: payment.id)
```

Comment:
`update_columns` bypasses all Validations and Callbacks etc. Why do you use this?